divi/src/main.cpp

```diff
@@ -104,8 +104,6 @@ static void CheckBlockIndex();
 /** Constant stuff for coinbase transactions we create: */
 CScript COINBASE_FLAGS;
 
-const string strMessageMagic = "DarkNet Signed Message:\n";
-
```

Comment:
This is a nonce for signature verification. Removing this would break signature verifications for important subcomponents of the system. MUST revert this change